Assets/Scripts/CutsceneCamera.cs

```diff
@@ -0,0 +1,46 @@
+using JetBrains.Annotations;
+using UnityEngine;
+using Yarn.Unity;
+
+public class CutsceneCamera : MonoBehaviour
+{
+    [CanBeNull] private static GameObject _targetObject;
+    private static Vector2 _target;
+
+    [SerializeField] private float xSmoothTime;
+    [SerializeField] private float ySmoothTime;
+    private float _xVelocity;
+    private float _yVelocity;
+
+    private void LateUpdate()
+    {
+        if (_targetObject) _target = _targetObject.transform.position;
+
+        // apply smoothing to the camera
```

Comment:
A lot of CutsceneCamera (especially these 4 lines) is duplicated with FollowCamera, but the differences are significant enough that it seems cleaner to have a little duplication than to try extracting a Camera superclass. Should keep in mind for future refactors- we might want a CameraUtility static class or something to avoid the duplication.

Reply:
that did not show the section of code i wanted oh well